src/migrations/20230306103400-add-project-column-to-segments.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `ALTER TABLE segments ADD COLUMN IF NOT EXISTS segment_project_id VARCHAR(255);`,
+        cb,
+    );
+    cb();
+};
+
+exports.down = function (db, cb) {
+    cb();
```

Comment:
Shouldn't we have a migration down here as well?

Reply:
Good point, we really should

Reply:
We have [many migrations without a down migration](https://sourcegraph.com/search?q=context%3Aglobal+repo%3A%5Egithub%5C.com%2FUnleash%2Funleash%24+cb%28%29&patternType=standard&sm=1&groupBy=path). In this case, adding a column shouldn't be harmful (although tests have shown that the wrong column name might cause issues), but dropping the column might unlink the segment from the project (which might not be that bad).

But, all in all, it doesn't hurt to add it if you consider it valuable. Particularly, I don't see much risk in not doing the down migration, unless we remove IF NOT EXISTS as suggested [here](https://github.com/Unleash/unleash/pull/3263#discussion_r1127731408)

Reply:
Added and removed `IF NOT EXISTS`